libs/blocks/aside/aside.js

```diff
@@ -127,8 +167,11 @@ function decoratePromobar(el) {
     const actionArea = child.querySelectorAll('em a, strong a, p > a strong');
     if (iconArea) textBlocks.shift();
     if (actionArea.length) textBlocks.pop();
-    if (textBlocks.length) combineTextBocks(textBlocks, iconArea, viewports[index]);
+    if (!(textBlocks.length || iconArea || actionArea.length)) return child.classList.add('hide-block');
+    if (textBlocks.length) combineTextBocks(textBlocks, iconArea, viewports[index], variant);
+    return true;
```

Comment:
What's the reason behind this return?

Reply:
@narcis-radu eslint is throwing error and lint check is failing in PR if there is no return in arrow function hence it was added.

Reply:
yes, that's because you introduced `return child.classList.add('hide-block');` on line 170. Is that needed?
```suggestion
  if (!(textBlocks.length || iconArea || actionArea.length)) {
    child.classList.add('hide-block');
    return;
  }
```

Reply:
@narcis-radu @chrischrischris I have removed the extra return at the end of the forEach block. The last if can be an else to the if (!(textBlocks.length || iconArea || actionArea.length)) { and return has been removed from that as well.